control/runtimes/paseo/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub mod runtime;
```

Comment:
Why are you `subxt codegen` for the paseo runtime, instead of the derive macro used in the other runtimes?

For example Westend:
```
#[subxt::subxt(
    runtime_metadata_path = "polkadot-metadata.bin",
    derive_for_all_types = "Clone"
)]
mod runtime {}

pub use runtime::*;
```

I dislike `subxt codegen` as it creates huge files that slow down our repo. whereas the macro does the codegen on the fly locally.

Reply:
Agreed, https://github.com/Snowfork/snowbridge/pull/1295/commits/02a73350a047b2c70b335550ec2390a51fb1bd6e.